xbmc/FileItem.h

```diff
@@ -499,6 +499,11 @@ class CFileItem :
    */
   std::string GetBaseMoviePath(bool useFolderNames) const;
 
+  /*! \brief Check to see if there is a 'Disc n' subdirectory in the movie path
+   \return the position of the start of the string if there is a 'Disc n' subdirectory in the movie path, 0 if not
```

Comment:
Now that we support Disc, Disk and localized version according to Kodi language settings, you need to extend this I guess.